install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -380,6 +380,34 @@ MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCyOJ5garOYw0sm
 ...
 -----END PRIVATE KEY-----
 ----
+
+- The secured registry should contain the following Subject Alternative Names (SAN) list:
++
+* Two service hostnames:
++
+----
+docker-registry.default.svc.cluster.local
+docker-registry.default.svc 
+----
++
+* service IP address. Use the following command to get the Docker registry service IP address:
```

Comment:
Looks like there's examples above, but this isn't showing an example. It's showing how to find what you need to put into the file. Is that correct? If yes, I'd suggest consistency, whether through how to find the example or the example itself. This goes for the below as well.